### DISCUSSION TITLE
How to use SEH registers

### DISCUSSION THREAD

**Discussion by 0xF6** [icedland/iced#548] [Q&A]
I'm trying to use Protected Mode Exceptions registers
As sample code:
```asm
mov rax, seh_handler
mov fs:[0], rax
```

I'il trying write
```C#
var @catch = asm.CreateLabel("seh_handler");
...
asm.mov(rax, __[@catch]);
asm.mov(fs, rax);
...
```
This does not seem to be the correct use, for example, I cannot specify the index where to write the address of the SEH handler procedure
the question is, how to use these registers correctly?

**Comment by wtfsck** [comment]
There's an example in the README:

https://github.com/icedland/iced/blob/master/src/rust/iced-x86/README.md#assemble-instructions

```rust
let _ = ptr(rax).fs(); // fs:[rax]
```
